### PR TITLE
[gh-pages][ZEPPELIN-2055] Add "What's new in 0.7" and Twitter feed to website main page

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### What is this PR for?
There are lots of improvement in upcoming release 0.7(as mentioned in #1972 (release note)). To emphasize them to users and make contents enrich Zeppelin official website, I added some description \w gif images. And also added Twitter news feed at the bottom of the main page. 

### What type of PR is it?
gh-pages | Improvement

### Todos
* [x] - need to take care of gif images' alignment when screen width is small 

### What is the Jira issue?
[ZEPPELIN-2055](https://issues.apache.org/jira/browse/ZEPPELIN-2055)

### How should this be tested?
Build `gh-pages` branch as described in [here](https://github.com/apache/zeppelin/tree/gh-pages#build-website). Or see the below gif sreenshot img :) 

### Screenshots (if appropriate)
 - Before 
before

 - After 
after

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Looks great!

What do you think add things about improvements on Python/pyspark in what's new section?

Matplotlib integration in python/pyspark and conda support in python are something worth to mention IMO.

**Comment by felixcheung** [comment]
^^ that should be a per release what's new page, I think?

File: _includes/themes/zeppelin/_navigation.html [hunk 1]
```diff
@@ -70,9 +70,13 @@
     <p>A web-based notebook that enables interactive data analytics. <br/>
       You can make beautiful data-driven, interactive and collaborative documents with SQL, Scala and more.
     </p>
-    <p><a href="http://youtu.be/_PQbVH_aO5E" target="_zeppelinVideo" class="btn btn-primary btn-lg bigFingerButton" role="button">Watch the video</a>
-
-       <a href="./download.html" class="btn btn-primary btn-lg bigFingerButton" role="button">Get Zeppelin</a></p>
+    <div class="border row">
+      <div class="col-md-12 col-sm-12 col-xs-12 text-center newBtn">
+        <a href="http://youtu.be/_PQbVH_aO5E" target="_zeppelinVideo" class="btn btn-primary btn-lg bigFingerButton" role="button">Tutorial Video</a>
+        <a href="./download.html" class="btn btn-primary btn-lg bigFingerButton" role="button">GET ZEPPELIN 0.7</a>
```
Comment (felixcheung) [inline _includes/themes/zeppelin/_navigation.html]: this is nice - should it just say "GET LATEST ZEPPELIN NOW"? so we don't have to keep updating this ;)

File: index.md [hunk 2]
```diff
@@ -18,10 +18,15 @@ limitations under the License.
 -->
 {% include JB/setup %}
 <br />
+<h1 class="index-header">
+  What is Apache Zeppelin ?
```
Comment (felixcheung) [inline index.md]: do we need the extra ` ` behind Zeppelin

**Comment by felixcheung** [comment]
looks good!

**Comment by AhyoungRyu** [comment]
@Leemoonsoo @felixcheung Thanks for your review! As @Leemoonsoo mentioned I added "Improvement in Python" section like below. How about this?
screen shot 2017-02-05 at 1 47 32 pm

@felixcheung Addressed your comments as well. And it'll be better to update "What's new ~ " section for every major release. Surely I'm willing to do that :)
screen shot 2017-02-05 at 2 28 44 pm

**Comment by felixcheung** [comment]
cool!

**Comment by Leemoonsoo** [comment]
@AhyoungRyu Great!

nit:

While this will be seen from not only existing Zeppelin user but also people new to Zeppelin, how about change "Interpreter Per User / Note" to something like "Multi-user support improvement", more general words. And address "Interpreter per user /note" and "interpreter impersonation" in the text below?

And in "Personalized Note Mode", would it be possible to somehow mention 'Collaboration' mode, although it's not really a new feature? Something like "Switch between Collaboration and Personalized mode". So people new to Zeppelin can easily understand that there're two possible modes switchable "Collaboration" and "Personalized". What do you think?

**Comment by AhyoungRyu** [comment]
@Leemoonsoo Thanks for the detailed suggestion. So I added some description about "Interpreter impersonation" under "Multi-user Support Improvement" and mentioned about "Collaboration mode"   . How about this? 
screen shot 2017-02-05 at 9 25 04 pm

Feel free to suggest if you have better wording or sentences in your mind :)

**Comment by Leemoonsoo** [comment]
LGTM!

**Comment by felixcheung** [comment]
LGTM

**Comment by AhyoungRyu** [comment]
Will merge into `gh-pages` if there are no more comments!

**Pull request closed without merging**